frontend/plugins/usePagePlugin.ts

```diff
@@ -279,16 +342,49 @@ function getPageCreateInput(input: PageDataCreateInput): CreatePageInput {
   };
 }
 
-function getMenuInput(data: MenuData): UpdateMenuInput {
+function getFooterInput(data: GlobalData): UpdateGlobalInput {
```

Comment:
`Footer` is a view concern that should interfere with data... Here we're not getting "footer" input, we're getting "global settings" input